### ISSUE TITLE
Support for GPU

### ISSUE THREAD

**Issue by davidbreitgand** [5g-media/incubator-openwhisk#1]
Add new convention <action-kind-name>@gpu meaning that this action will run on a node with GPU which was previously exposed as a schedulable resouce  in the K8s cluster according to this specification: https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/

**Comment by gpu** [comment]
Yeah, :+1: for supporting me! (You cannot imagine how many GitHub notifications I'm already receiving due to my username ;-) )
